frontend/src/metabase/search/components/filters/TypeFilter/TypeFilterContent.tsx

```diff
@@ -7,9 +7,10 @@ import { SearchFilterPopoverWrapper } from "metabase/search/components/SearchFil
 import { enabledSearchTypes } from "metabase/search/constants";
 import type { SearchFilterDropdown } from "metabase/search/types";
 import { Checkbox, Stack } from "metabase/ui";
-import type { EnabledSearchModel } from "metabase-types/api";
+import type { EnabledSearchModel, SearchContext } from "metabase-types/api";
 
 const EMPTY_SEARCH_QUERY = {
+  context: "search-model-filter" as SearchContext,
```

Comment:
Let's type `EMPTY_SEARCH_QUERY: SearchRequest` and then we won't need a type cast

```suggestion
  context: "search-model-filter",
```